### DISCUSSION TITLE
How to remove numeric validation for year in bib?

### DISCUSSION THREAD

**Discussion by cstby** [alshedivat/al-folio#2491] [Q&A] [answered]
My understanding is that `forthcoming` is a perfectly valid value for the  `year` field of a .bib citation. Right now, my build fails when entering `forthcoming` as a value. How can I remove this validation?

**Comment by george-gca** [comment, marked as answer]
Year is expected to be a number, so the publications are sorted and the most recent appears first. This is done in:

https://github.com/alshedivat/al-folio/blob/1bee4d152a7d0dc2a3a2e501cc089dd006690a1f/_config.yml#L324
